innopoints/schemas/variety.py

```diff
@@ -58,6 +65,7 @@ def wire_color_size(self, data, **kwargs):  # pylint: disable=unused-argument
 
     @pre_load
     def enumerate_images(self, data, **kwargs):  # pylint: disable=unused-argument
+        """Convert the array of URL strings to an array of image objects with order."""
         if self.context.get('update', False):
             if 'images' in data:
                 data['images'] = [{'order': idx, 'image_id': int(url.split('/')[2])}
```

Comment:
Better use `[-1]` to get the last element (the id), so if we ever want to implement nested namespaces (or a namespace simply has a '/' in its name) it will still work

Reply:
yeah, but for now those are forbidden, so it's best to fall with an error than to accept anything malicious. For now we have no plans of implementing this, so I see no reason to support it.

Reply:
the thing is that it will not fail with an error. If you get "folder1/26/43", it will use 26 as the id instead of 43

Reply:
This URL will not even make it past Flask (I think so anyway)